source/index.html

```diff
@@ -29,7 +29,7 @@ <h1>
         <h1>Recent Blog Posts</h1>
 
         {% for post in site.posts limit: 4 %}
-        <li class="post" style="display: grid; font-size: 16px;">
+        <li class="post" style="display: grid; font-size: 16px">
```

Comment:
Styles should end with ;?

Reply:
that's some prettier shit I removed.

Reply:
Yeah already resolved it, started the review on mobile and moved over, apparently it stored a partial review 🤷‍♂️